packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -38,13 +41,13 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   }
 
   const getCurrentCardAccount = function* (cardId) {
-    const eligibleAccounts = yield select(selectors.components.debitCard.getEligibleAccounts)
+    const eligibleAccountsR = yield select(selectors.components.debitCard.getEligibleAccounts)
     try {
       yield put(A.getCurrentCardAccountLoading())
 
       const { accountCurrency } = yield call(api.getDCCurrentAccount, cardId)
 
-      const accountFound = findAccount(accountCurrency, eligibleAccounts)
+      const accountFound = findAccount(accountCurrency, eligibleAccountsR.data)
```

Comment:
You can't use the `data` prop on remote because it might not be there if state is not Success

Reply:
If you have it on state, pass it into the getCurrentCardAccount action, or a 2nd option is to use `getOrElse`